src/sentry/tasks/store.py

```diff
@@ -657,6 +657,10 @@ def save_event(
     time_limit=65,
     soft_time_limit=60,
     silo_mode=SiloMode.REGION,
+    taskworker_config=TaskworkerConfig(
+        namespace=ingest_transactions_tasks,
+        processing_deadline_duration=65,
+    ),
 )
 def save_event_transaction(
```

Comment:
Do we know if `save_event_transaction` gets called with extra `kwargs` ever? And if so, that might be a potential vector for un-serializable objects.

Reply:
Good point. No, it doesn't seem like it. The only other reference is here: https://github.com/getsentry/sentry/blob/270022cea134ce833b25488d893fd514a5ee4f34/src/sentry/ingest/consumer/processors.py#L245-L251